plugin/error_vis.py

```diff
@@ -184,6 +218,63 @@ def _as_html(errors_dict):
         return errors_html
 
     @staticmethod
+    def _as_phantom_html(errors_dict):
+        """Get error as html for phantom.
+
+        Args:
+            errors_dict (dict): Current error
+        """
+        stylesheet = '''
+            <style>
```

Comment:
Not needed to be changed here, but do you think we would benefit from it if we would have style templates as html in some folder of the plugin? Then there will be no need to hardcode them here.